### PR TITLE
[AWS S3] Improvements

### PR DESCRIPTION
**Pull request by shiftedreality**
## Purpose of this pull request

Created new topic for remote storage migration.

* https://jira.corp.magento.com/browse/MC-38902
* https://jira.corp.magento.com/browse/MC-38905

## Affected DevDocs pages

-  ...

## Links to Magento source code

-  ...

### CODE REVIEW

File: src/guides/v2.4/config-guide/remote-storage/config-remote-storage-aws-s3.md [hunk 1]
```diff
@@ -22,10 +22,31 @@ Prerequisites for using the AWS S3 adapter:
...
+    proxy_hide_header "x-amz-request-id";
+    proxy_hide_header "x-amz-storage-class";
+    proxy_hide_header "Set-Cookie";
+    proxy_ignore_headers "Set-Cookie";
+}
+```
+
+If you're using access and secret keys instead of [AWS IAM] roles, you'll need a separate module.
+
+We recommend using the [`ngx_aws_auth`][ngx repo].
```
Comment (hguthrie) [inline src/guides/v2.4/config-guide/remote-storage/config-remote-storage-aws-s3.md:46]: Is this the separate module they have to use as indicated in the previous sentence?

```suggestion
We recommend using the [`ngx_aws_auth`][ngx repo] Nginx module.
```

**Review by hguthrie — CHANGES_REQUESTED** [review]

File: src/guides/v2.4/config-guide/remote-storage/config-remote-storage-migration.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+group: configuration-guide
+title: Migrate to remote storage
+functional_areas:
+  - Configuration
+  - Remote Storage
+---
+
```
[the remark this answers is not in the document]
Comment (shiftedreality) [reply src/guides/v2.4/config-guide/remote-storage/config-remote-storage-migration.md:8]: Migration itself does not move but copies content. So, no additional preparation needed

File: src/guides/v2.4/config-guide/remote-storage/config-remote-storage.md [hunk 3]
```diff
@@ -46,11 +46,11 @@ The following table lists the parameters available for configuring the storage a
 
 ## Configure image resizing
 
-Magento supports image resizing on the application side by default. With the Remote Storage module enabled, you can offload image resizing to the server (Nginx) side, which is a simple and efficient way to save disk resources and optimize disk usage.
+Magento supports image resizing on the application side by default. With the Remote Storage module enabled, you can offload an image resizing to the server (Nginx) side, which is a simple and efficient way to save disk resources and optimize disk usage.
```
[the remark this answers is not in the document]
Comment (shiftedreality) [reply src/guides/v2.4/config-guide/remote-storage/config-remote-storage.md:49]: it should be action

File: src/guides/v2.4/config-guide/remote-storage/config-remote-storage.md [hunk 4]
```diff
@@ -27,10 +27,10 @@ The default storage location is the local filesystem. A _storage adapter_ allows
 
 -  [Amazon Simple Storage Service (Amazon S3)][AWS S3]
 
-The 'remote-storage' command in the Magento CLI enables the remote storage module and allows you to set the storage service parameters. Minimally, you must supply the storage driver, the object storage name, and the storage location. For example, to enable AWS S3 remote storage:
+The `setup:config:set` command in the Magento CLI enables the remote storage module and allows you to set the storage service parameters. Minimally, you must supply the storage driver, the object storage name, and the storage location. For example, to enable AWS S3 remote storage:
 
 ```bash
-bin/magento remote-storage:enable aws-s3 region-name bucket-name [optional prefix] --access-key=<optional-access-key> --secret-key=<optional-secret-key>
+bin/magento setup:config:set --remote-storage-driver="<driver-name>" --remote-storage-bucket="<bucket-name>" --remote-storage-region="<region-name>" --remote-storage-prefix="<optional-prefix>" --access-key=<optional-access-key> --secret-key=<optional-secret-key> -n
```
[the remark this answers is not in the document]
Comment (shiftedreality) [reply src/guides/v2.4/config-guide/remote-storage/config-remote-storage.md:33]: yes, only driver and bucket aree required

**Review by hguthrie — APPROVED** [review]

**Pull request merged**